web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -109,9 +109,9 @@ export class FirebaseDataConverter {
     return new Form(
       id,
       Map<string, Field>(
-        keys(data.elements).map((id: string) => [
+        keys(data.fields).map((id: string) => [
```

Comment:
Was this changed by accident? In the current spec it's still called `elements`:
https://github.com/google/ground-platform/wiki/Cloud-Firestore-Representation#document-structure

Reply:
No its not a mistake, toForm method is being passed fields key instead of elements, nobody noticed that earlier

Reply:
In our current data model "fields" are a special type of form element. In the schema there currently isn't a property with the name `fields`, only `elements`. If we're accidentally saving it to Firebase with the name "fields" elsewhere we should update it to "elements" so we all use the same database doc structure.